embulk-jruby-strptime/src/main/java/org/embulk/spi/time/StrptimeFormat.java

```diff
@@ -0,0 +1,44 @@
+package org.embulk.spi.time;
+
+// Ported from org.jruby.util.RubyDateFormatter.Format in JRuby
```

Comment:
Since JRuby has EPL license, I believe that the porting should be ok. And I added notations on licensing according to your review comments. d8c4956cbd189e2fed2632eb403f0dee37a9c014